.github/workflows/test-docs-build.yml

```diff
@@ -21,8 +21,23 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
-      - name: Test Build
+
+      - name: Test Wiki Build
         # TODO Add 'yarn build' as last line once broken links are fixed
         run: |
           cd documentation
           yarn install --immutable
+
+      - name: Install stable toolchain
+        uses: actions-rs/toolchain@v1
+        with:
+          toolchain: stable
+          override: true
+
+      - name: Test Rust Documentation
+        uses: actions-rs/cargo@v1
+        env:
+          RUSTDOCFLAGS: "-D warnings"
+        with:
+          command: doc
+          args: --all-features --no-deps --workspace
```

Comment:
Can we build it the way docs.rs builds the documentation, so we don't run into the `0.6` doc problems again?
So specifically using nightly and `--cfg docsrs`. So:

```
RUSTDOCFLAGS='-D warnings --cfg docsrs' cargo +nightly doc --all-features --no-deps --workspace
```

Reply:
Good point, will do.

Is the `docsrs` config really necessary in our code though?
For example, the double `cfg` features in `identty_iota`?

```rust
#[cfg(feature = "account")]
#[cfg_attr(docsrs, doc(cfg(feature = "account")))]
pub mod account_storage {
```

Reply:
The double-cfg is necessary I think, since `#[doc(cfg(...))]` is unstable. It's how crypto.rs does it, too. I mainly copied their approach: https://github.com/iotaledger/crypto.rs/blob/94be7913f11018467edb64c3d126ab682c29edcf/src/signatures/mod.rs#L5.

Yes, I think `all-features = true` in the toml metadata should be fine.